coops-ui/src/ts/enums/entityTypes.ts

```diff
@@ -0,0 +1,4 @@
+export enum EntityTypes {
```

Comment:
Severin: I see there is a 'ts' folder under 'src' now. I don't think we need to differentiate between source code types. Was this discussed as a team and agreed to? I will go along if the team(s) think this is the way to go but otherwise I think we can just have src/enums, src/interfaces, etc. Thoughts?

Cameron: I'm open to ideas, imo i thought was a good idea to consolidate our enums, constants and interfaces opposed to them living individually scattered throughout components, we can certainly name the root folder for these values anything we like.

Severin: No, I'm just asking to remove the /ts folder level. By all means let's use enums/, interfaces/ etc to keep those specific components together.

Reply:
Would you recommend another name for a directory opposed to just having all the enums, constants, interfaces  living in the root level of src?

Reply:
I'm thinking of this:
/coops-ui
   /src
      /asssets
      /components
      /enums
      /interfaces
      /mixins
      /store
etc

Reply:
Alrighty!